### DISCUSSION TITLE
Discoverability of GitHub Sponsors Profiles

### DISCUSSION THREAD

**Discussion by terrytangyuan** [community/community#3717] [Other Feature Feedback, Questions, & Ideas]
Thanks to GitHub team for building such an awesome product!

I wonder if there's any plan to make the profiles more discoverable. I ask because I never saw my profile listed on [this page](https://github.com/sponsors/community) under “Explore->GitHub Sponsors” even though I opted it in my settings. 

I believe there are many developers who are in great demand of sponsorships/funding but fail to promote their work/projects on social media successfully to get the attention they deserve. This is especially the case for minorities and people in under-represented groups.

**Comment by bdresser** [comment]
@terrytangyuan thank you for asking! This is super important and something we're actively thinking about. The Sponsors community page is a good start, but it's somewhat limited, and we're looking at a couple small improvements to help potential sponsors find folks they might be interested in.

I'm curious: how do you promote your work or find Sponsors? What's been effective for you in growing your program? And beyond the Sponsors community page you linked, how could GitHub help you find the audience that might be interested in your work?

**Reply by terrytangyuan** [reply]
Thanks for the response! I am glad this is something GitHub team is actively thinking about. So far I've only tried sharing link to my sponsors page on social media, e.g. [on Twitter](https://twitter.com/TerryTangYuan):
* First tweet on this when I finished setting up my sponsors page [link](https://twitter.com/TerryTangYuan/status/1196833967439257600?s=20)
* Second tweet on this when I shared the news that custom amounts and one-time payments are available [link](https://twitter.com/TerryTangYuan/status/1379512104630226951?s=20)

However it's not very effective since I don't use Twitter very often and not influential there.

Besides this, I also posted the link to my sponsors page in my [GitHub profile README](https://github.com/terrytangyuan/terrytangyuan) but have no way to track whether there's any traffic directly from there. I'd imagine a traffic graph for the sponsors page similar to that of GitHub repos would be useful.

Hopefully this would be useful and I look forward to see what's coming next for GitHub Sponsors! Feel free to reach out personally if I can be helpful for testing beta features, etc.
